test/libsolidity/semanticTests/revertStrings/enum_v2.sol

```diff
@@ -0,0 +1,13 @@
+pragma abicoder v2;
+contract C {
+	enum E {X, Y}
+	function f(E[] calldata arr) external {
+		arr[1];
+	}
+}
+// ====
```

Comment:
Why an extra test?

Reply:
This one runs with abicoder v2 while the existing one was meant for v1.